src/kriscv/__main__.py

```diff
@@ -115,6 +127,16 @@ def _addr_to_hex(addr: int) -> str:
             out.write(word + '\n')
 
 
+def _kriscv_assemble(opts: AssembleOpts) -> None:
```

Comment:
This should be moved into its own module and exposed under a separate command, e.g. `kriscv-asm` or `kriscv-dev asm`.